### PR TITLE
REX-5771 Add points as payment source.

### PR DESCRIPTION
**Pull request by ketanp77**
Ticket: https://eonx.atlassian.net/browse/REX-5771

### CODE REVIEW

File: src/Traits/PaymentSources/PointsTrait.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?php
+declare(strict_types=1);
+
+namespace EoneoPay\PhpSdk\Traits\PaymentSources;
+
+use Symfony\Component\Serializer\Annotation\Groups;
+
+trait PointsTrait
```
Comment (natepage) [inline src/Traits/PaymentSources/PointsTrait.php:8]: @ketanp77 Looking at the requests sent by Jose in his tests, I can't see `externalId`, and we're missing `api_key` and `points`.

```json
"payment_source": {
    "account_id": "<account_id>",
    "provider_id": "<provider_id>",
    "api_key": "<api_key>",
    "points": 46,
    "type": "points"
  },
```

**Review by natepage — CHANGES_REQUESTED** [review]

File: src/Traits/PaymentSources/PointsTrait.php (hunk 1 above)
Comment (ketanp77) [reply src/Traits/PaymentSources/PointsTrait.php:8]: He didn't send me those ...I will add 'em

Comment (ketanp77) [reply src/Traits/PaymentSources/PointsTrait.php:8]: Fixed.

**Review by natepage — APPROVED** [review]
@ketanp77 Thank you kind sir!

**Pull request merged**